1.Bitcoin'in iyi kazandırdığı 2017 yılı sonunda yatırım yaptığınızı farz edelim. Günlük ortalama %12 artan bitcoine 1000$ yatırmış olsaydınız bir hafta sonunda paranız ne kadar olurdu? Lütfen problemi Python kullanarak çözün.

In [14]:
ana_para = 1000

In [10]:
gunluk_faiz = 12


In [11]:
carpan = (100+ gunluk_faiz)/100
print (carpan)

1.12


In [12]:
sure = 7 #1 hafta sonunda

In [16]:
son_miktar = ana_para * carpan ** sure 
print(son_miktar)

2210.6814074060812


In [18]:
kar = son_miktar - ana_para
print(kar)

1210.6814074060812


In [7]:
yatirim_araci = "bitcoin"

2.Tırnak içindeki metni Python ile yazdırın. Ancak rakamlar değişkenlerden gelmeli. Metin uzun olduğu için iki satıra yazmayı düşünebilirsiniz: "Hafta başında 1000 dolarlık bitcoin aldığımızda günde ortalama %12 kazançla, bir hafta sonunda 1210.68 dolar kazanırdık"

In [19]:
print ("""Hafta başında {} dolarlık bitcoin aldığımızda günde ortalama %{} kazançla, 
bir hafta sonunda {:.2f} dolar kazanırdık.""".format(ana_para, gunluk_faiz, kar))

Hafta başında 1000 dolarlık bitcoin aldığımızda günde ortalama %12 kazançla, 
bir hafta sonunda 1210.68 dolar kazanırdık.


3.Kullanıcıdan dosya adını alan ve bu dosya adına "py" uzantısı ekleyerek ekrana yazdıran bir kod yazınız.

Kullanıcıdan dosya adını alan ve bu dosya adına "py" uzantısı ekleyerek ekrana yazdıran bir kod yazınız.

In [20]:
dosya_adi = input ( 'dosya :') 
print ( "dosya {} .py".format(dosya_adi)) 

dosya :dosya
dosya dosya .py
